composer.json

```diff
@@ -37,13 +36,17 @@
         "sonata-project/formatter-bundle": "~2.2",
         "sonata-project/datagrid-bundle": "~2.2",
         "sonata-project/seo-bundle": "^1.1.5",
+        "amazonwebservices/aws-sdk-for-php": "~1.6",
```

Comment:
I need to add this to require-dev since it's required to pass the tests. other way they will fail or being skipped.